CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+Contributing
+============
+
+Contributions are encouraged from anyone! Every little bit helps, and credit will always be given.
+Please see the contributing guidelines in [the zlux repository](https://github.com/zowe/zlux/blob/master/CONTRIBUTING.md)
```

Comment:
Though this probably won't change, should probably point to https://github.com/zowe/zlux/blob/v2.x/staging/CONTRIBUTING.md